.eslintrc.json

```diff
@@ -11,7 +12,88 @@
         "allowSingleLine": true
       }
     ],
+    "comma-spacing": "error",
+    "comma-style": [
+      "error",
+      "last"
+    ],
+    "curly": "error",
+    "eol-last": "error",
+    "eqeqeq": [
+      "error",
+      "smart"
```

Comment:
There's `== null` in `cli.js` and I'm not sure if I can get rid of that safely...